samples/web-workers/ts/polyfill.worker.ts

```diff
@@ -0,0 +1,19 @@
+import { JSDOM } from "jsdom";
+
+let { window } = new JSDOM();
+
+if (!globalThis.document) {
```

Comment:
Haven't done much of these polyfills before, so let me know how evil this is 😄 and if you have any suggestions